### ISSUE TITLE
Error when installing from pip on Windows

### ISSUE THREAD

**Issue by gahaalt** [bogliosimone/similaripy#2]
Hello, thank you for making this package.
I get following logs, when trying to install similaripy with pip on Windows. Can you help me?
```

 pip install similaripy
Collecting similaripy
  Using cached https://files.pythonhosted.org/packages/9a/3a/44821be2e597c49b9bc5bd066df3729893af54c6209d7b05cafe3d39e448/similaripy-0.1.1.tar.gz
Requirement already satisfied: scipy>=1.0.0 in c:\users\gaha\anaconda3\lib\site-packages (from similaripy) (1.3.2)
Requirement already satisfied: numpy>=1.14.0 in c:\users\gaha\anaconda3\lib\site-packages (from similaripy) (1.17.4)
Requirement already satisfied: tqdm>=4.19.6 in c:\users\gaha\anaconda3\lib\site-packages (from similaripy) (4.40.2)
Building wheels for collected packages: similaripy
  Building wheel for similaripy (setup.py) ... |  WARNING: Subprocess output does not appear to be encoded as cp1250
error
  ERROR: Command errored out with exit status 1:
   command: 'C:\Users\gaha\Anaconda3\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\gaha\\AppData\\Local\\Temp\\pip-install-midhwich\\similaripy\\setup.py'"'"'; __file__='"'"'C:\\Users\\gaha\\AppData\\Local\\Temp\\pip-install-midhwich\\similaripy\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\gaha\AppData\Local\Temp\pip-wheel-_z119i4r' --python-tag cp37
       cwd: C:\Users\gaha\AppData\Local\Temp\pip-install-midhwich\similaripy\
  Complete output (38 lines):
  C:\Users\gaha\Anaconda3\lib\site-packages\setuptools\dist.py:476: UserWarning: Normalizing '0.1.01' to '0.1.1'
    normalized_version,
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.7
  creating build\lib.win-amd64-3.7\similaripy
  copying similaripy\normalization.py -> build\lib.win-amd64-3.7\similaripy
  copying similaripy\similarity.py -> build\lib.win-amd64-3.7\similaripy
  copying similaripy\utility.py -> build\lib.win-amd64-3.7\similaripy
  copying similaripy\__init__.py -> build\lib.win-amd64-3.7\similaripy
  running egg_info
  writing similaripy.egg-info\PKG-INFO
  writing dependency_links to similaripy.egg-info\dependency_links.txt
  writing requirements to similaripy.egg-info\requires.txt
  writing top-level names to similaripy.egg-info\top_level.txt
  reading manifest file 'similaripy.egg-info\SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  writing manifest file 'similaripy.egg-info\SOURCES.txt'
  creating build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\__init__.py -> build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\coo_to_csr.h -> build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\normalization.cpp -> build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\normalization.pyx -> build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\s_plus.cpp -> build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\s_plus.h -> build\lib.win-amd64-3.7\similaripy\cython_code
  copying similaripy\cython_code\s_plus.pyx -> build\lib.win-amd64-3.7\similaripy\cython_code
  running build_ext
  building 'similaripy.cython_code.s_plus' extension
  creating build\temp.win-amd64-3.7
  creating build\temp.win-amd64-3.7\Release
  creating build\temp.win-amd64-3.7\Release\similaripy
  creating build\temp.win-amd64-3.7\Release\similaripy\cython_code
  C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -Isimilaripy\cython_code -IC:\Users\gaha\Anaconda3\include -IC:\Users\gaha\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include" /EHsc /Tpsimilaripy\cython_code\s_plus.cpp /Fobuild\temp.win-amd64-3.7\Release\similaripy\cython_code\s_plus.obj /O2 /openmp
  s_plus.cpp
  C:\Users\gaha\Anaconda3\include\pyconfig.h(59): fatal error C1083: Nie moľna otworzy† pliku do\x88Ącz: 'io.h': No such file or directory
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.23.28105\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
  ----------------------------------------
  ERROR: Failed building wheel for similaripy
  Running setup.py clean for similaripy
Failed to build similaripy
Installing collected packages: similaripy
    Running setup.py install for similaripy ... \    WARNING: Subprocess output does not appear to be encoded as cp1250
error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\gaha\Anaconda3\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\gaha\\AppData\\Local\\Temp\\pip-install-midhwich\\similaripy\\setup.py'"'"'; __file__='"'"'C:\\Users\\gaha\\AppData\\Local\\Temp\\pip-install-midhwich\\similaripy\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\gaha\AppData\Local\Temp\pip-record-gdm22cde\install-record.txt' --single-version-externally-managed --compile
         cwd: C:\Users\gaha\AppData\Local\Temp\pip-install-midhwich\similaripy\
    Complete output (38 lines):
    C:\Users\gaha\Anaconda3\lib\site-packages\setuptools\dist.py:476: UserWarning: Normalizing '0.1.01' to '0.1.1'
      normalized_version,
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.7
    creating build\lib.win-amd64-3.7\similaripy
    copying similaripy\normalization.py -> build\lib.win-amd64-3.7\similaripy
    copying similaripy\similarity.py -> build\lib.win-amd64-3.7\similaripy
    copying similaripy\utility.py -> build\lib.win-amd64-3.7\similaripy
    copying similaripy\__init__.py -> build\lib.win-amd64-3.7\similaripy
    running egg_info
    writing similaripy.egg-info\PKG-INFO
    writing dependency_links to similaripy.egg-info\dependency_links.txt
    writing requirements to similaripy.egg-info\requires.txt
    writing top-level names to similaripy.egg-info\top_level.txt
    reading manifest file 'similaripy.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'similaripy.egg-info\SOURCES.txt'
    creating build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\__init__.py -> build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\coo_to_csr.h -> build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\normalization.cpp -> build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\normalization.pyx -> build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\s_plus.cpp -> build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\s_plus.h -> build\lib.win-amd64-3.7\similaripy\cython_code
    copying similaripy\cython_code\s_plus.pyx -> build\lib.win-amd64-3.7\similaripy\cython_code
    running build_ext
    building 'similaripy.cython_code.s_plus' extension
    creating build\temp.win-amd64-3.7
    creating build\temp.win-amd64-3.7\Release
    creating build\temp.win-amd64-3.7\Release\similaripy
    creating build\temp.win-amd64-3.7\Release\similaripy\cython_code
    C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -Isimilaripy\cython_code -IC:\Users\gaha\Anaconda3\include -IC:\Users\gaha\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\include" /EHsc /Tpsimilaripy\cython_code\s_plus.cpp /Fobuild\temp.win-amd64-3.7\Release\similaripy\cython_code\s_plus.obj /O2 /openmp
    s_plus.cpp
    C:\Users\gaha\Anaconda3\include\pyconfig.h(59): fatal error C1083: Nie moľna otworzy† pliku do\x88Ącz: 'io.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.23.28105\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\gaha\Anaconda3\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\gaha\\AppData\\Local\\Temp\\pip-install-midhwich\\similaripy\\setup.py'"'"'; __file__='"'"'C:\\Users\\gaha\\AppData\\Local\\Temp\\pip-install-midhwich\\similaripy\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\gaha\AppData\Local\Temp\pip-record-gdm22cde\install-record.txt' --single-version-externally-managed --compile Check the logs for full command output. for full command output.
```

**Comment by bogliosimone** [comment]
Hi @gahaalt,
it seems the problem is related to the file _io.h_

Unfortunately on my Windows installation I cannot reproduce the error (all the cython files are compiled correctly without errors).

I try to google this problem and I found some interesting topic ([here](https://stackoverflow.com/questions/40018405/cannot-open-include-file-io-h-no-such-file-or-directory) one as example)

I saw that some users solved this problem uninstalling and reinstalling Visual Studio.

Let me know if you find a solution, maybe it can be helpful for some other users.

Thanks
Simone
